openmc/tallies.py

```diff
@@ -13,11 +13,12 @@
 
 import numpy as np
 
-from openmc import Mesh, Filter, Trigger, Nuclide
-from openmc.arithmetic import *
+from openmc import Filter, Trigger, Nuclide
+from openmc.arithmetic import CrossScore, CrossNuclide, CrossFilter, \
```

Comment:
This is on place I have to differ with pylint / PEP8 - if you import / use *everything* class contained within a module, it doesn't seem efficient to enumerate each item like this. But alas, let's play by the rules.